README.md

```diff
@@ -5,6 +5,7 @@
 
 - **Dedicated repository:** [rust-lang/wg-governance][repository]
 - **Discord channel:** [#wg-governance][discord]
+- **Dedicated meetings document:** [https://hackmd.io/ATj1rZJaRimaIfIWfAOYfQ?edit]
```

Comment:
What do you think of calling it like this instead?
```suggestion
- **Meeting Agenda:** [https://hackmd.io/ATj1rZJaRimaIfIWfAOYfQ?edit]
```

Reply:
works for me.